build-tools/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suppressions PUBLIC
-    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
-    "http://www.puppycrawl.com/dtds/suppressions_1_0.dtd">
+    "-//Checkstyle//DTD Suppressions 1.3//EN"
+    "http://checkstyle.sourceforge.net/dtds/suppressions_1_0.dtd">
```

Comment:
Seems 1.1 is the latest version, or we keep use 1.0?
<!DOCTYPE suppressions PUBLIC
    "-//Checkstyle//DTD Suppressions 1.1//EN"
    "http://checkstyle.sourceforge.net/dtds/suppressions_1_1.dtd">